resolwe/flow/tests/test_url_file.py

```diff
@@ -0,0 +1,51 @@
+# pylint: disable=missing-docstring
+from resolwe.flow.models import Process
+from resolwe.test import ProcessTestCase, tag_process
+
+
+class URLFileTest(ProcessTestCase):
+    @tag_process('url-file-process')
```

Comment:
Please move the process to the [`processes` directory](https://github.com/genialis/resolwe/tree/master/resolwe/flow/tests/processes). 
Check here for an example how to use it: https://github.com/genialis/resolwe/blob/master/resolwe/flow/tests/test_executors.py#L63